test/drbgtest.c

```diff
@@ -667,8 +680,10 @@ static int test_drbg_reseed(int expect_success,
 
     /* Generate random output from the public and private DRBG */
     before_reseed = expect_master_reseed == 1 ? reseed_when : 0;
-    if (!TEST_int_eq(RAND_bytes(buf, sizeof(buf)), expect_success)
-        || !TEST_int_eq(RAND_priv_bytes(buf, sizeof(buf)), expect_success))
+    if (!TEST_int_eq(RAND_bytes((unsigned char*)private_random,
+                                RANDOM_SIZE), expect_success)
+        || !TEST_int_eq(RAND_priv_bytes((unsigned char*) public_random,
```

Comment:
Yes thanks, that's a mistake. I was undecided for a while whether I should test the private drbg first (because it's more important), but abandoned that idea. Somehow I only swapped the calls back but forgot about the buffers.